packages/contracts-periphery/contracts/universal/drippie/Drippie.sol

```diff
@@ -201,6 +213,29 @@ contract Drippie is AssetReceiver {
             "Drippie: dripcheck failed so drip is not yet ready to be triggered"
         );
 
+        // Alright, we're good to execute.
+        return true;
+    }
+
+    /**
+     * Triggers a drip. This function is deliberately left as a public function because the
+     * assumption being made here is that setting the drip to ACTIVE is an affirmative signal that
+     * the drip should be executable according to the drip parameters, drip check, and drip
+     * interval. Note that drip parameters are read entirely from the state and are not supplied as
+     * user input, so there should not be any way for a non-authorized user to influence the
+     * behavior of the drip.
+     *
+     * @param _name Name of the drip to trigger.
+     */
+    function drip(string memory _name) external {
+        DripState storage state = drips[_name];
```

Comment:
Do you need to `sload` a bunch here before the `require` statement?

Reply:
No only SLOADs when things actually get used